coda-automation/.circleci/config.yml

```diff
@@ -0,0 +1,222 @@
+# ---
```

Comment:
Pretty sure all of the CircleCI stuff is super CRUFT so should be ok to remove/not worth migrating.